playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml

```diff
@@ -43,23 +51,26 @@
       expect:
         command: "sftp admin@{{ hostvars[item].nodepool.private_ipv4 }}"
         responses:
-          'password:': admin
+          '(?i)password:': admin123
           'sftp>':
             - put id_rsa.pub.b64
             - exit
         timeout: 60
+      when: item not in ["iosxr-7.0.2"]
```

Comment:
lets try not to make this dynamic. if the password is the only issue, lets respin

Reply:
lets drop when statement